### PR TITLE
feat: add chr() function to yaml

### PR DESCRIPTION
**Pull request by zhixingheyi-tian**
Hi @jvanstraten @jacques-n 
Please give a review, Thanks.

### CODE REVIEW

File: extensions/functions_string.yaml [hunk 1]
```diff
@@ -17,6 +17,27 @@ scalar_functions:
         variadic:
           min: 1
         return: "string"
+  - name: chr
+    description: >-
+    Returns the ASCII character having the binary equivalent to input.
+    If n is larger than 256 the result is equivalent to chr(n % 256)
```
Comment (richtia) [inline extensions/functions_string.yaml:23]: what's the behavior for negative values of n?

Comment (zhixingheyi-tian) [reply extensions/functions_string.yaml]: Add description for negative values.

**Comment by zhixingheyi-tian** [comment]
Hi @richtia 
Can it be merged？

**Review by richtia — COMMENTED** [review]
I'm thinking about how we should handle multi-byte character sets, such as unicode.

Some engines seem to provide an options for the chr function (oracle), while other have a separate nchr/nchar function (sql server).

@westonpace do you have any thoughts on this?

File: extensions/functions_string.yaml [hunk 2]
```diff
@@ -17,6 +17,28 @@ scalar_functions:
...
+  - name: chr
+    description: >-
+    Returns the ASCII character having the binary equivalent to input.
+    If n is larger than 256 the result is equivalent to chr(n % 256).
+    If n is negative value, the result is null.
+    impls:
+    - args:
+      - value: i8
+        name: "input"
+      return: "fixedchar<L1>"
```
Comment (westonpace) [inline extensions/functions_string.yaml:29]: What is `L1` here?  None of the input arguments are parameterized so I don't think the output can be parameterized.  Do you perhaps want `fixedchar<1>`?  It seems postgres, duckdb, datafusion, and presto all return a variable size string.  Only SQL server returns a fixed size string.  I think returning `string` here would be more consistent with existing implementations.

**Review by westonpace — CHANGES_REQUESTED** [review]

**Comment by jacques-n** [comment]
No progress has been made in more than six months. Closing without prejudice.

**Pull request closed without merging**